app/mailers/candidate_mailer.rb

```diff
@@ -295,6 +295,18 @@ def offer_accepted(application_choice)
     )
   end
 
+  def ucas_match_initial_email_duplicate_applications(application_choice)
+    @application_form = application_choice.application_form
+    @course_name_and_code = application_choice.course_option.course.name_and_code
+    @provider_name = application_choice.course_option.course.provider.name
+    @date_to_withdraw_application_by = 10.business_days.after(Time.zone.today).to_s(:govuk_date)
```

Comment:
It would be good to put this rule somewhere closer to the data — is there a way we could push it up into the model? How do we track the date these initial emails were sent?

Reply:
We update `candidate_last contacted_at` and `action taken`. I'm not sure if this can be automated by using these fields, as `candidate_last_contacted_at` can potentially be mapped to other type of contacts as well, and I assume that the support agent verifies when the last set of emails were sent through the audit log before sending out the next round of emails. 

I can however add another field to map this. @AgaDufrat can you confirm if this is the case, and do you have any thoughts on maybe added another field?